presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -94,6 +95,9 @@ public String explain(CreateTable statement, List<Expression> parameters)
                 if ((type == null) || type.equals(UNKNOWN)) {
                     throw new SemanticException(TYPE_MISMATCH, column, "Unknown type for column '%s' ", column.getName());
                 }
+                if (columns.stream().anyMatch(c -> c.getName().equals(column.getName()))) {
+                    throw new SemanticException(DUPLICATE_COLUMN_NAME, column, "Column name '%s' specified more than once", column.getName());
+                }
                 columns.add(new ColumnMetadata(column.getName(), type, column.getComment().orElse(null), false));
```

Comment:
The case of `element instanceof LikeClause` also adds columns. Shouldn't that add be covered with a check too?